src/style.css

```diff
@@ -0,0 +1,120 @@
+/* stylelint-disable declaration-block-trailing-semicolon */
+/* stylelint-disable shorthand-property-no-redundant-values */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
why do we need to reset margins and paddings for `*` selector?

Reply:
body and html don`t reset margins and paddings